### ISSUE TITLE
-rpc-enable on windows connection refused

### ISSUE THREAD

**Issue by fredshao** [XDagger/xdag#387]
`xdag -rpc-enable -> wallet exit directly`

`xdag -m 0 xdag.uupool.cn:13654 -rpc-enable -> runing`

```
 curl -X POST --data '{"jsonrpc":"2.0","method":"xdag_version","params":[]," id":23}' -H "Content-Type: application/json" http://127.0.0.1:7677
curl: (7) Failed to connect to 127.0.0.1 port 7677: Connection refused
```

**Comment by xrdavies** [comment]
@fredshao  You have run it like this.
./xdag xdag.uupool.cn:13654 -rpc-enable -rpc-port 7677

**Comment by fredshao** [comment]
![image](https://user-images.githubusercontent.com/23414494/47475675-29cb3780-d84f-11e8-9ee6-78fd77ac6b0b.png)

**Comment by xrdavies** [comment]
already fixed by @jonano614 in 0.3.1
